src/events/components/CalendarView/calendar.less

```diff
@@ -26,6 +26,20 @@
   height: 100px;
   background: #fff;
 }
+.tileToday {
+  &:extend(.tile);

```

Comment:
Since the html element already has the class "tile" I think this would just lead to duplicated CSS?